src/__tests__/actions/loginAction/__test__/socialAction.test.js

```diff
@@ -35,4 +35,12 @@ describe("Tests facebook actions", () => {
     store.dispatch(socialAction({ access: "token" }, "facebook"));
     expect(store.getActions()).toEqual([]); 
   });
+
+
+
+
+
```

Comment:
@CryceTruly What are all these blank lines for???

Reply:
Thanks, @joelethan, thanks for pointing out this eslint issue. I have refactored it.